### PR TITLE
[6.0] feature: use laravel's forceIndex implementation

### PR DESCRIPTION
**Pull request by taka-oyama**
Fix: #112 

## Checklist

- [x] CHANGELOG

### CODE REVIEW

File: tests/Query/BuilderTest.php [hunk 1]
```diff
@@ -431,17 +431,39 @@ public function testForceIndex(): void
...
+    }
+
+    public function test_ignoreIndex(): void
+    {
+        $this->expectExceptionMessage('Cloud Spanner does not support index type: ignore');
+        $this->expectException(BadMethodCallException::class);
+
+        $this->getDefaultConnection()
+            ->table(self::TABLE_NAME_USER)
+            ->useIndex('test_index_name2')
```
Comment (zeriyoshi) [inline tests/Query/BuilderTest.php:463]:
```suggestion
            ->ignoreIndex('test_index_name2')
```

**Review by zeriyoshi — CHANGES_REQUESTED** [review]
Please check suggestion

**Review by zeriyoshi — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# v6.0.0 [Not released Yet]
+
+Changed
+- [Breaking] Match `Quer\Builder::forceIndex()` behavior with laravel's (`forceIndex` property no longer exists). (#114)
```
Comment (t-matsuno-777) [inline CHANGELOG.md:4]:
```suggestion
- [Breaking] Match `Query\Builder::forceIndex()` behavior with laravel's (`forceIndex` property no longer exists). (#114)
```

**Review by t-matsuno-777 — APPROVED** [review]

**Pull request merged**